src/resources/views/crud/create.blade.php

```diff
@@ -53,3 +53,7 @@
 </div>
 
 @endsection
+
+@section('after_scripts')
```

Comment:
This stack will be fowarded into on the fly specific files because modal content is not on page on first render so no longer have access to regular crud stacks when we load it. If a modal loads a field that was not previously loaded by parent crud we were not able to push the crud scripts and assets to the crud stack on first render. 

I had the briliant idea of stack fowarding with the same name, so in the field when you use @push('crud_field_scripts') it will either push to the real stack or the one created with same name for on the fly fields. :bulb: ahaha